users/jwt/idp.go

```diff
@@ -45,27 +45,29 @@ func (idp *jwtIdentityProvider) TemporaryKey(id string) (string, error) {
 	return idp.jwt(claims)
 }
 
-func (idp *jwtIdentityProvider) Identity(key string) (string, error) {
+func (idp *jwtIdentityProvider) Identity(key string) (string, errors.Error) {
 	token, err := jwt.Parse(key, func(token *jwt.Token) (interface{}, error) {
 		if _, ok := token.Method.(*jwt.SigningMethodHMAC); !ok {
 			return nil, users.ErrUnauthorizedAccess
 		}
 
-		return []byte(idp.secret), nil
+		return []byte(idp.secret), errors.Empty()
 	})
 
 	if err != nil {
```

Comment:
You're comparing to `nil`, but errors.Empty() is not an error, but it's not `nil`, either, so it will fall under this case.